src/components/Bio.jsx

```diff
@@ -0,0 +1,42 @@
+import JohnGuitar from "../../public/jh_guitar.png";
+import Target from "../../public/target2.svg";
+
+const Bio = () => {
+  return (
```

Comment:
You can turn this into an explicit return since theres no logic. 


`Const Bio = () => (
   <div>...code</div
)`